<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>marker</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon/ladybug.png</normaloff>icon/ladybug.png</iconset>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>icon/back.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="feedback2">
   <property name="geometry">
    <rect>
     <x>885</x>
     <y>450</y>
     <width>189</width>
     <height>189</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { 
   background-image : url(icon/ladybug.png); 
   border : 0px;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="feedback3">
   <property name="geometry">
    <rect>
     <x>1550</x>
     <y>450</y>
     <width>189</width>
     <height>189</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { 
   background-image : url(icon/ladybug.png); 
   border : 0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="feedback1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>450</y>
     <width>189</width>
     <height>189</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { 
   background-image : url(icon/ladybug.png); 
   border : 0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="replay">
   <property name="geometry">
    <rect>
     <x>1625</x>
     <y>922</y>
     <width>225</width>
     <height>225</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { 
   background-image : url(icon/replay.png); 
   border : 0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="home">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>922</y>
     <width>189</width>
     <height>189</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { 
   background-image : url(icon/home.png); 
   border : 0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
